style.css

```diff
@@ -1,489 +1,772 @@
 * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-    --gold-color: #d2b429;
-    --header-font: 'Cormorant Upright', 'Georgia', serif;
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	--gold-color: #d2b429;
+	--header-font: "Old Standard TT", "Georgia", serif;
 }
 
 body {
-    background-color: black;
-    margin: 0;
-    font-family: 'Montserrat', sans-serif;
-    font-weight: 400;
-    font-size: 20px;
-    color: white;
+	background-color: black;
+	margin: 0;
+	font-family: "Montserrat", sans-serif;
+	font-weight: 400;
+	font-size: 20px;
+	color: white;
 }
 
 .clear {
-    clear: both;
+	clear: both;
 }
 
 .text-black {
-    color: #000;
-    line-height: 2.1rem;
+	color: #000;
+	line-height: 2.1rem;
 }
 
 p.text-white {
-    color: #fff;
-    line-height: 2.1rem;
-    padding-bottom: 2rem;
+	color: #fff;
+	line-height: 2.1rem;
+	padding-bottom: 2rem;
 }
 
 .uppercase {
-    text-transform: uppercase;
+	text-transform: uppercase;
 }
 
 button {
-    background-color: #fff;
-    color: var(--gold-color);
-    font-family: var(--header-font);
-    text-align: center;
-    border: solid 1px var(--gold-color);
-    font-size: 2rem;
+	background-color: #fff;
+	color: var(--gold-color);
+	font-family: var(--header-font);
+	text-align: center;
+	border: solid 1px var(--gold-color);
+	font-size: 2rem;
 }
 
 p.button-text {
-    padding: 1rem;
+	padding: 1rem;
 }
 
 .section-row {
-    max-width: 71rem;
-    min-height: 130px;
-    display: flex;
-    position: relative;
-    margin: auto;
+	max-width: 71rem;
+	min-height: 130px;
+	display: flex;
+	position: relative;
+	margin: auto;
 }
 
 h3.black {
-    color: #000;
-    font-size: 2.5rem;
-    font-family: var(--header-font);
-    text-align: center;
+	color: #000;
+	font-size: 2.5rem;
+	font-family: var(--header-font);
+	text-align: center;
 }
 
 h3.white {
-    color: #fff;
-    font-size: 2.5rem;
-    font-family: var(--header-font);
-    text-align: center;
-    padding-bottom: 2rem;
+	color: #fff;
+	font-size: 2.5rem;
+	font-family: var(--header-font);
+	text-align: center;
+	padding-bottom: 2rem;
 }
 
 /*---Nav---*/
 
 nav.menu {
-    max-width: 1140px;
-    min-height: 130px;
-    position: relative;
-    z-index: 999;
-    display: flex;
-    margin-right: auto;
-    margin-left: auto;
+	max-width: 1140px;
+	min-height: 130px;
+	position: relative;
+	z-index: 999;
```

Comment:
Do we need such a high z-index value? It's basically best to use as low z-index value as possible, probably 1 or 2 would be fine in this case. It's much easier to handle z-axis then, especially when multiple items appear in one stacking context.

A little more reading on the topic: https://www.joshwcomeau.com/css/stacking-contexts/

Reply:
Done.